lib/Cmts.ml

```diff
@@ -110,6 +110,18 @@ let find_at_position t loc pos =
     Holds if there is only whitespace between the locations, or if there is a
     [|] character and the first location begins a line and the start column
     of the first location is lower than that of the second location. *)
```

Comment:
This comment was attached to `is_adjacent`, I think it'll be better to leave it there and to add a comment for the new function.
Also, I think `is_adjacent_prev` could be implemented as an optional argument to `is_adjacent` (the filter function)